odh-dashboard/base/odh-dashboard-crd.yaml

```diff
@@ -45,6 +45,8 @@ spec:
                       type: boolean
                     disableModelServing:
                       type: boolean
+                    modelMetricsNamespace:
```

Comment:
We don't actually use this in RHODS and even if we did the feature is hardcoded to be disabled since we tried to add it in 1.24 all the way up to the current date.

https://github.com/opendatahub-io/odh-dashboard/issues/1128 will cover the re-addition.